examples/simple-page/bemhtml-templates.js

```diff
@@ -0,0 +1,179 @@
+module.exports = function() {
+  // 1. Common HTML elements
+  // — doctype
+  // — html, head, meta, title, body
+  block('page')(
```

Comment:
I'd refactor this template to:
1. Get rid of `<section>` provided by `page` — I think it's kinda hard code instead of a good example of reusable block.
2. Get rid of extra elements (`html`, `head`, 'meta-utf', 'title')
3. Move `__title`'s content to BEMTREE
4. Provide possibility to customize `lang` attribute (or remove it for simplicity but do not hard code anyway).
5. Make `vcard` explicit `mix` so it'd be possible to show that `attrs` belong to `vcard` and not to `page__body`